digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/linearasset/TextValueLinearAssetService.scala

```diff
@@ -122,4 +122,15 @@ class TextValueLinearAssetService(roadLinkServiceImpl: RoadLinkService, eventBus
       }
     }
   }
+
+  override def validateAssetValue(values: Option[Value]): Unit = {
+    val euroPattern = "^[0-9|Ee][0-9|Bb]{0,2}"
```

Comment:
Since this function is on TextualService this name could be change, to not be just about the european ones